### ISSUE TITLE
DynamoDB permission

### ISSUE THREAD

**Issue by eversonl** [chr8993/places-skill#2]
Can you update the Readme to detail the permissions required on DynamoDB please?
Whatever I try I keep getting "message": "User: arn:aws:sts::9930863xx:assumed-role/lambda_basic_execution/places is not authorized to perform: dynamodb:GetItem on resource: arn:aws:dynamodb:eu-west-1:9930863xxx:table/alexaStorage",

I've been at this one for hours haha

**Comment by chr8993** [comment]
Sure thing, check out the updated Readme. Basically, you just need to attach the `AmazonDynamoDBFullAccess` policy to the role you attached to your lambda function in the IAM console. This allows for the creation of the table.

**Comment by eversonl** [comment]
Thanks, I think I cracked that just after posting, what should the Key be?
I'm sorry for all the questions, I am searching and trying to figure this out.

**Comment by chr8993** [comment]
Are you referring to the Google API Key?

**Comment by eversonl** [comment]
No, sorry, when creating the Dynamo table its asking for a key. I have tried a few options however, I am receiving the error : Error fetching user state: ValidationException: The provided key element does not match the schema" in the cloudwatch logs and when testing via the Lambda test function

**Comment by chr8993** [comment]
Hmm thats strange.. It should automatically create the table, but I believe the key is automatically set to `userId`

**Comment by eversonl** [comment]
OK, thats me getting ahead of myself. I deleted the table and ran the Lambda test a couple more times and the table created. Interestingly the Lambda Management Console test feature is still complaining with the above error however the Developer console is responding.

**Comment by chr8993** [comment]
Have you tried adding the DynamoDb trigger to your Lambda function? This can be updated within the triggers tab where you manage your Lambda function.
